src/main/java/org/jenkinsci/plugins/registry/notification/EnvContributor.java

```diff
@@ -55,6 +61,22 @@ public void buildEnvironmentFor(@Nonnull Run r, @Nonnull EnvVars envs, @Nonnull
             for (ParameterValue parameter : parameters) {
                 parameter.buildEnvironment(r, envs);
             }
+            final Job parent = r.getParent();
+            if (parent instanceof ParameterizedJobMixIn.ParameterizedJob) {
+                final DockerHubTrigger trigger = DockerHubTrigger.getTrigger((ParameterizedJobMixIn.ParameterizedJob)parent);
+                if (trigger != null) {
+                    final List<EventType> eventTypes = trigger.getEventTypes();
+                    if (eventTypes != null) {
+                        final String dtrJsonType = cause.getPushNotification().getDtrEventJSONTypeEventJSONType();
+                        for (EventType type : eventTypes) {
+                            if (type.accepts(dtrJsonType)) {
```

Comment:
Is this handling a potential `null` here?

Reply:
Above mentioned null handling in the trigger makes this unnecessary - removing.